# Hexapod robot leg tip positions and body position/orientation given in global frame
geometry_msgs/Point[6] leg_position
geometry_msgs/Pose body_pose
